katcp/test/test_server.py

```diff
@@ -886,6 +881,41 @@ def test_handler_exceptions(self):
              ""),
         ])
 
+    def test_decorated_handler(self):
+        """Test handling with @request and @return_reply."""
+
+        stringy = "sss"
```

Comment:
mmhhh, perhaps instead of all the `...y`, rather use `_string`, `_float` and `_int`

> 'Readability counts' - Time Peters